src/System.Reflection.MetadataLoadContext/tests/src/FromCoreFx/TempDirectory.cs

```diff
@@ -0,0 +1,46 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.IO
```

Comment:
This "FromCoreFX" folder can hopefully be eliminated now in favor of the actual CoreFX files.

Reply:
Yes. Thanks